domain/schema/model.go

```diff
@@ -29,3 +30,17 @@ CREATE TABLE model_config (
 );
 `)
 }
+
+func spacesSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE spaces (
+    uuid            TEXT PRIMARY KEY,
+    name            TEXT NOT NULL,
+    is_public       BOOLEAN,
+    provider_uuid   TEXT
```

Comment:
Should this be a foreign key to somewhere? If not can we add a comment about it?

Reply:
this should not be a fk, I added a comment